src/__test__/components.test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import store from '../redux/store';
+import Contact from '../components/Pages/Contact';
+import Details from '../components/Pages/Details';
+import Home from '../components/Pages/Home';
+import '@testing-library/jest-dom';
+
+describe('Testing Profile and Mission Components', () => {
+  it('Testing MissionContainer components', () => {
+    const home = render(
+      <React.StrictMode>
+        <Router>
+          <Provider store={store}>
+            <Home />
+          </Provider>
+        </Router>
+      </React.StrictMode>,
+    );
+    expect(home).toMatchSnapshot();
+  });
+
+  it('Testing MissionContainer components', () => {
+    const contact = render(
+      <React.StrictMode>
+        <Router>
+          <Provider store={store}>
+            <Contact />
+          </Provider>
+        </Router>
+      </React.StrictMode>,
+    );
+    expect(contact).toMatchSnapshot();
+  });
+
+  it('Testing MissionContainer components', () => {
+    const details = render(
+      <React.StrictMode>
+        <Router>
+          <Provider store={store}>
+            <Details />
+          </Provider>
+        </Router>
+      </React.StrictMode>,
+    );
+    expect(details).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] As requested of you by the previous reviewer and as it is in the project requirements, you are required to Create unit tests for pure functions in your Redux code. As I can currently see, none of this has been handled. Kindly write unit tests for your Redux methods.

![image](https://user-images.githubusercontent.com/86318284/204351962-941d7278-d42f-492d-9994-6b64ba944ca5.png)